### PR TITLE
Add multi timer support

### PR DESCRIPTION
**Pull request by robsdedude**
Proposing a new feature: having multiple timer/notice groups. Each having their own settings and timer running.
This also adds the option to shuffle the messages in groups and send them at random times (upper and lower limit needs to be specified; uniformly distributed).

I'll mark this as WIP because I only tested everything briefly. I'll deploy it and remove the WIP state when I've successfully used it for a while. Feedback is very welcome already.

It looks like this:
image
or on a smaller screen:
image

### CODE REVIEW

**Comment by robsdedude** [comment]
I *think* this is ready to be merged now. I've used it for quite a while now and so far my bot did not catch fire. **But** this is a rather big change (+1337 lines, `nice`). Please test this feature yourself again before merging it.

File: javascript-source/core/updates.js [hunk 1]
```diff
@@ -228,6 +229,66 @@
         $.inidb.set('updates', 'installedv3.5.0', 'true');
     }
 
+    /* version 3.4.7 updates */
```
Comment (robsdedude) [inline javascript-source/core/updates.js:232]: a) I just realized the version in the comment is not what the following code says.
b) Will this feature go into 3.5 anyway?

Comment (gmt2001) [reply javascript-source/core/updates.js:232]: Yes

The update should still go off anyway, but if you want you can move it to the 3.5.0 block

**Review by gmt2001 — APPROVED** [review]

**Comment by gmt2001** [comment]
closes gmt2001/PhantomBot-1#148

**Comment by robsdedude** [comment]
:partying_face:

**Pull request merged**